src/SUMMARY.md

```diff
@@ -89,10 +90,6 @@
   - [Deriving](methods-and-traits/deriving.md)
   - [Exercise: Generic Logger](methods-and-traits/exercise.md)
     - [Solution](methods-and-traits/solution.md)
-
-# Day 2: Afternoon
-
-- [Welcome](welcome-day-2-afternoon.md)
```

Comment:
Why is this being moved?

Reply:
oops!, It's by mistake. I will revert it.